### ISSUE TITLE
Text ist truncated if longer than browser window

### ISSUE THREAD

**Issue by whoot** [EvotecIT/PSWriteHTML#35]
If only one DataTable is used, the inserted text is truncated if it is longer than the browser window.
In this case, it would be a good if the text simply continued in a new line.

Example - Only one DataTable but text gets cutted at browser end:
![grafik](https://user-images.githubusercontent.com/8066322/64126446-101f4880-cdae-11e9-9cbe-29a561201ae9.png)

**Comment by PrzemyslawKlys** [comment]
Do you actually have a legitimate reason for such long text in one line without any spaces?

```powershell
Import-Module .\PSWriteHTML.psd1 -Force

$Time = Start-TimeLog

$ReportTitle = 'Test'

$DomainAdminTable = Get-ADForest | Select-Object ForestMode, Name, RootDomain, SchemaMaster

$ImageLink = 'https://evotec.xyz/wp-content/uploads/2015/05/Logo-evotec-012.png'

New-HTML -TitleText $ReportTitle -UseJavaScriptLinks -UseCssLinks -FilePath "$PSScriptRoot\Example3.html" {
    New-HTMLLogo -RightLogoString $ImageLink
    New-HTMLTab -TabName 'Dashboard' {
        New-HTMLContent  -HeaderText "Groups" {
            New-HTMLContent -HeaderText 'Domain Administrators' -CanCollapse {
                New-HTMLPanel {
                    New-HTMLTable -DataTable $DomainAdminTable -HideFooter
                }
            }
        }
        New-HTMLSection {

            $Data = @(
                'string dfdsfsdfsd fsdf sdf dsf sdfsdf sdf fsd fsd fsdf sdf sdf sdlfksd fsdjkf hsdfk hsdfks hfksdh fkshdf sdf sdf fs dfsdfdsfs dfsdfsdfsdfsdfsdsdfsdf sdfsdf sdfsf sdf fsdfsd fsdf sdfsdf sdfs dfsdfsd f'
                'string dfdsfsdfsd fsdf sdf dsf sdfsdf sdf fsd fsd fsdf sdf sdf sdlfksd fsdjkf hsdfk hsdfks hfksdh fkshdf sdf sdf fs dfsdfdsfs dfsdfsdfsdfsdfsdsdfsdf sdfsdf sdfsf sdf fsdfsd fsdf sdfsdf sdfs dfsdfsd f'
            )

            New-HTMLTable -DataTable $Data
        }
        New-HTMLSection {

            $Data = 'string dfdsfsdfsd fsdf sdf dsf sdfsdf sdf fsd fsd fsdf sdf sdf sdlfksd fsdjkf hsdfk hsdfks hfksdh fkshdf sdf sdf fs dfsdfdsfs dfsdfsdfsdfsdfsdsdfsdf sdfsdf sdfsf sdf fsdfsd fsdf sdfsdf sdfs dfsdfsd f string dfdsfsdfsd fsdf sdf dsf sdfsdf sdf fsd fsd fsdf sdf sdf sdlfksd fsdjkf hsdfk hsdfks hfksdh fkshdf sdf sdf fs dfsdfdsfs dfsdfsdfsdfsdfsdsdfsdf sdfsdf sdfsf sdf fsdfsd fsdf sdfsdf sdfs dfsdfsd f'

            New-HTMLTable -DataTable $Data
        }
        New-HTMLSection {

            $Data = 'stringfdggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggdfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffdddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddfffffffffffffffffffffffffffffffffffffffffffff'

            New-HTMLTable -DataTable $Data
        }
    }
    New-HTMLTab -TabName 'Something' {

    }
    New-HTMLTab -TabName 'This is very long title' {

    }
    New-HTMLTab -TabName 'This is long title' {

    }
} -ShowHTML
```

This seems to work fine for text that has spaces.

**Comment by PrzemyslawKlys** [comment]
This seems like an easy fix with:

```css
td {
  word-break: break-all;
}
```

Not sure if there's a catch to this.

**Comment by whoot** [comment]
> Do you actually have a legitimate reason for such long text in one line without any spaces?

Yes, I want to put hash values in tables.

**Comment by PrzemyslawKlys** [comment]
Can you do some testing by adding above word-break into CSS and see if everything works? As in the thing you want works, the thing that works still works?

**Comment by PrzemyslawKlys** [comment]
One alternative to this would be to use ScrollX feature instead of Responsive. You could also use what I've shown here: https://github.com/EvotecIT/PSWriteHTML/issues/32 basically hiding hash columns under responsive button by using ResponsiveOperations.

**Comment by whoot** [comment]
How can I use this in Dashimo?

Im using something like this:
```
Dashboard -Name 'Example Dashboard' -FilePath $PSScriptRoot\LongText.html -Show {
	Tab -Name 'Example' {
		Section -Name 'Super Long Text' {
			Table -DataTable $superLongText -HideFooter
		}
	}
```

**Comment by PrzemyslawKlys** [comment]
Open up longtext.html with text editor and put it somewhere in the beginning. For example between hr / default section

![image](https://user-images.githubusercontent.com/15063294/64130646-f12bb100-cdc3-11e9-94b0-382f31dee8ff.png)

**Comment by whoot** [comment]
Thanks! This fixes the issue!

**Comment by PrzemyslawKlys** [comment]
Play with it for a while and let me know. Maybe it will impact negatively things. I'll add if I won't hear from you and have more time to test things on my end.

**Comment by whoot** [comment]
It indeed "breaks" some stuff. I don't know what criteria determines the column sizes, but sometimes one column is too wide and could be made smaller, while in another culoumn the text gets truncated.

Example:
![grafik](https://user-images.githubusercontent.com/8066322/64132470-60a79d80-cdd0-11e9-9cd9-974612133180.png)

**Comment by PrzemyslawKlys** [comment]
Ye, like i said. Please use workarounds I've given in linked issue. Should look nice.

**Comment by PrzemyslawKlys** [comment]
Please check #107 for a final fix. Will be available as part of 0.0.75.
